game/system/newpad.cpp

```diff
@@ -43,6 +47,37 @@ u64 input_mode_mod = 0;
 u64 input_mode_index = 0;
 MappingInfo g_input_mode_mapping;
 
+MappingInfo g_mapping;  // Local copy of Gfx mapping info
```

Comment:
I remember it was a little awkward in the beginning using the mapping in g_settings for analog settings. Mostly because I wasn't familiar with this part of the code base at the time. I'll see if I can use g_settings now that I'm more familiar with the code